### ISSUE TITLE
docker run fails on npx keycloakify start-keycloak

### ISSUE THREAD

**Issue by Assaro** [keycloakify/keycloakify#618]
I tried starting keycloakify in a docker container for the first time and got the follwing error:
```bash
$ docker run \
    -p 8080:8080 \
    --name keycloak-keycloakify \
    -e KEYCLOAK_ADMIN=admin \
    -e KEYCLOAK_ADMIN_PASSWORD=admin \
    -v ".\node_modules\.cache\keycloakify\myrealm-realm-25.json":/opt/keycloak/data/import/myrealm-realm.json \
    -v "./node_modules\.cache\keycloakify\keycloak-theme-for-kc-22-and-above.jar":/opt/keycloak/providers/keycloak-theme.jar \
    -v ".\dist_keycloak\theme\keycloakify-starter":/opt/keycloak/themes/keycloakify-starter:rw \
    quay.io/keycloak/keycloak:25.0.4 \
    start-dev \
    --import-realm
docker: Error response from daemon: ./node_modules\.cache\keycloakify\keycloak-theme-for-kc-22-and-above.jar%!(EXTRA string=is not a valid Windows path).
```
I think this is happening because the 2nd `-v` variable uses `./node_modules\` instead of `.\node_modules\`

I am using:
- W10 Enterprise 22H2
- Node.js 18.20.3
- Maven 3.9.9

It happens with both the webpack and the vite starter.

**Comment by garronej** [comment]
Ok thank you for reporting. I'll release a patch in a few minutes

**Comment by garronej** [comment]
@Assaro, it's fixed in 10.0.0-rc.146.  

Thank you for reporting
